googletest/src/gtest-internal-inl.h

```diff
@@ -290,7 +290,7 @@ inline int CountIf(const Container& c, Predicate predicate) {
   // Implemented as an explicit loop since std::count_if() in libCstd on
   // Solaris has a non-standard signature.
```

Comment:
Just wondering, would this workaround (a hand-written "count-if", below here) still be necessary, or is `std::count_if` nowadays fine on any supported platform? The workaround was introduced by zhanyong.wan, April 12, 2011 "Fixes Sun C++ compiler errors (by Pasi Valminen)", https://github.com/google/googletest/commit/b8c0e16eeb7496f71480c6a060144b0e050edcf5